docusaurus.config.js

```diff
@@ -65,14 +66,22 @@ const config = {
         },
         googleAnalytics: enableGoogleAnalytics
           ? {
-              trackingID: googleAnalyticsConfig.trackingID,
-              anonymizeIP: true,
-            }
+            trackingID: googleAnalyticsConfig.trackingID,
+            anonymizeIP: true,
+          }
           : undefined,
       },
     ],
   ],
 
+  // clientModules are custom modules (anything that Webpack can import), so css, js, etc
+  // that get included on every page and can be used to run arbitrary client-side Javascript, for example: 
+  clientModules: [path.resolve('./scripts/module-tutorial-client.js')],
+
+  customFields: {
+    tutorialServiceEndpoint: "https://h1t1m9ck07.execute-api.us-east-1.amazonaws.com/tutorial-service/post"
```

Comment:
Is this still being used somewhere?

Reply:
Ah great catch - thank you! Will remove.

Reply:
Removed